Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -877,20 +889,21 @@ JSGlobalContextRef WebFrame::jsContextForWorld(DOMWrapperWorld& world)
 
 JSGlobalContextRef WebFrame::jsContextForWorld(InjectedBundleScriptWorld* world)
 {
-    return jsContextForWorld(world->coreWorld());
+    return jsContextForWorld(RefPtr { world }->coreWorld());
```

Comment:
This isn't needed, `world` should be protected before it's passed into this function.